File: CP Lab/Lab6/2.c
// Lab project by Sarang Dev Saha (20UCS173)
//LAB4_QUES 2
/* Read two single digit numbers and calculate first number raised to the power
second number. */
# include <stdio.h>
# include <math.h> //using math.h header to accomodate pow() function
int main() {
    float a,b, number;
	//Taking the numbers from the user
	printf("Enter a single digit number: ");
	scanf("%f",&a);
	printf("Enter another single digit number: ");
	scanf("%f",&b);
	//using pow() function to calculate the power of the number.
    number= pow(a,b);
	printf("The resulting number is %f",number);
    return 0;
}

File: CP Lab/Lab3/2.py
# Lab Project by Sarang Dev Saha 

# LAB_03 QUE_02
#Read a number from keyboard and print it’s integral and fractional part separately

#Taking the input from the user
number=float(input("Enter a number:")) 
#Assigning a variable the value of integral part of the number
int_number=int(number)
#Assigning another variable the value of fractional part of the number
frac_number=float(number)-int(number)
print("The integral part of the number is",int_number)
print("The fractional part of the number is",frac_number)